specification/cognitiveservices/data-plane/LUIS/Runtime/v2.0/LUIS-Runtime.json

```diff
@@ -0,0 +1,338 @@
+          }
+        }
+      },
+      "post": {
+        "description": "Gets the published endpoint prediction for the given long query.",
+        "operationId": "Prediction_GetPredictionsFromEndpointViaPost",
+        "parameters": [
+          {
+            "name": "appId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "Format - guid. The application ID."
+          },
+          {
+            "name": "q",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Q"
+            },
+            "description": "The query to predict."
+          },
+          {
+            "name": "timezoneOffset",
+            "in": "query",
+            "description": "The timezone offset for the location of the request.",
+            "type": "number"
+          },
+          {
+            "name": "verbose",
+            "in": "query",
+            "description": "If true will return all intents instead of just the topscoring intent.",
+            "type": "boolean"
+          },
+          {
+            "name": "spellCheck",
+            "in": "query",
+            "description": "Enable spell checking.",
+            "type": "boolean"
+          },
+          {
+            "name": "staging",
+            "in": "query",
+            "description": "Use staging endpoint.",
+            "type": "boolean"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A JSON object containing the prediction.",
+            "schema": {
+              "$ref": "#/definitions/LuisResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples":{
+          "Successful Get Predictions From Endpoint Via Post request": {
+            "$ref": "./examples/prediction/SuccessfulGetPredictionsFromEndpointViaPostRequest.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Q": {
+      "type": "string",
+      "example": "\"forward to frank 30 dollars through HSBC\""
+    },
+    "LuisResult": {
+      "type": "object",
+      "properties": {
+        "query": {
+            "type": "string",
+            "x-nullable": true
+        },
+        "topScoringIntent": {
+          "$ref": "#/definitions/IntentModel",
+          "x-nullable": true
+        },
+        "intents": {
+          "type": "array",
+          "x-nullable": true,
+          "items": {
+            "$ref": "#/definitions/IntentModel"
+          }
+        },
+        "entities": {
+          "type": "array",
+          "x-nullable": true,
```

Comment:
Is there a reason you're explicitly marking things `x-nullable: true` ?

Reply:
Removed all x-nullable: true